src/Foundatio.RabbitMQ/Messaging/RabbitMQMessageBus.cs

```diff
@@ -117,21 +131,22 @@ public class RabbitMQMessageBus : MessageBusBase, IMessageBus {
                 return;
             _logger.Trace("Message Publish: {messageType}", messageType.FullName);
 
-            // RabbitMQ only supports delayed messages with a third party plugin.
-            if (delay.HasValue && delay.Value > TimeSpan.Zero) {
```

Comment:
I think this should still happen if you don't have _delayExchange set to true.. This will at least give you some kind of basic delayed message (as long as your app doesn't crash).